onedrive: DirMove failing with: The bound function binding to 'microsoft.graph.driveItem' does not support the escape function annotation.

#### What is the problem you are having with rclone?

The integration tests [are failing on onedrive](https://pub.rclone.org/integration-tests/current/onedrive-fs.operations-TestOneDrive-5.txt)

#### What is your rclone version (output from `rclone version`)

master, though this is reproducible with v1.49.5, v1.48, v1.47 (didn't try any further back)

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)

Ubuntu 19.04, amd64

####  Which cloud storage system are you using? (eg Google Drive)

OneDrive personal

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

```
$ cd fs/operations
$ go test -v -remote TestOneDrive: -verbose -test.run '^TestDirMove$'
2019/10/25 14:51:21 DEBUG : One drive root 'rclone-test-kurazes5xihixih2meqekeh7': Token expired but no uploads in progress - doing nothing
2019/10/25 14:51:21 DEBUG : TestOneDrive: Loaded invalid token from config file - ignoring
2019/10/25 14:51:21 DEBUG : TestOneDrive: Saved new token in config file
=== RUN   TestDirMove
2019/10/25 14:51:23 DEBUG : A1/one: Starting multipart upload
2019/10/25 14:51:23 DEBUG : A1/one: Uploading segment 0/3 size 3
2019/10/25 14:51:24 DEBUG : A1/two: Starting multipart upload
2019/10/25 14:51:24 DEBUG : A1/two: Uploading segment 0/3 size 3
2019/10/25 14:51:25 DEBUG : A1/B1/three: Starting multipart upload
2019/10/25 14:51:26 DEBUG : A1/B1/three: Uploading segment 0/5 size 5
2019/10/25 14:51:26 DEBUG : A1/B1/C1/four: Starting multipart upload
2019/10/25 14:51:27 DEBUG : A1/B1/C1/four: Uploading segment 0/4 size 4
2019/10/25 14:51:28 DEBUG : A1/B1/C2/five: Starting multipart upload
2019/10/25 14:51:28 DEBUG : A1/B1/C2/five: Uploading segment 0/4 size 4
2019/10/25 14:51:28 DEBUG : A1/B2: Making directory
2019/10/25 14:51:29 DEBUG : A1/B1/C3: Making directory
--- FAIL: TestDirMove (14.17s)
    run.go:176: Remote "One drive root 'rclone-test-kurazes5xihixih2meqekeh7'", Local "Local file system at /tmp/rclone678421448", Modify Window "1s"
    operations_test.go:1304: 
        	Error Trace:	operations_test.go:1304
        	Error:      	Received unexpected error:
        	            	BadRequest: The request URI is not valid. The bound function binding to 'microsoft.graph.driveItem' does not support the escape function annotation.
        	Test:       	TestDirMove
FAIL
2019/10/25 14:51:36 DEBUG : One drive root 'rclone-test-kurazes5xihixih2meqekeh7': Purge remote
exit status 1
FAIL	github.com/rclone/rclone/fs/operations	15.314s
```

@Cnly have you got time to take a look at this?

--- a/backend/onedrive/onedrive.go
+++ b/backend/onedrive/onedrive.go
@@ -351,8 +351,13 @@ func shouldRetry(resp *http.Response, err error) (bool, error) {
 // instead of simply using `drives/driveID/root:/itemPath` because it works for
 // "shared with me" folders in OneDrive Personal (See #2536, #2778)
 // This path pattern comes from https://github.com/OneDrive/onedrive-api-docs/issues/908#issuecomment-417488480
+//
+// If `relPath` == '', do not append the slash (See #3664)
 func (f *Fs) readMetaDataForPathRelativeToID(ctx context.Context, normalizedID string, relPath string) (info *api.Item, resp *http.Response, err error) {
-	opts := newOptsCall(normalizedID, "GET", ":/"+withTrailingColon(rest.URLPathEscape(enc.FromStandardPath(relPath))))
+	if relPath != "" {
+		relPath = "/" + withTrailingColon(rest.URLPathEscape(enc.FromStandardPath(relPath)))
+	}
+	opts := newOptsCall(normalizedID, "GET", ":"+relPath)
 	err = f.pacer.Call(func() (bool, error) {
 		resp, err = f.srv.CallJSON(ctx, &opts, nil, &info)
 		return shouldRetry(resp, err)

